cloud-api/src/main/java/com/sequenceiq/cloudbreak/cloud/event/context/CloudContext.java

```diff
@@ -1,51 +1,30 @@
 package com.sequenceiq.cloudbreak.cloud.event.context;
 
 import com.sequenceiq.cloudbreak.cloud.CloudPlatformVariant;
-import com.sequenceiq.cloudbreak.domain.AdjustmentType;
-import com.sequenceiq.cloudbreak.domain.OnFailureAction;
-import com.sequenceiq.cloudbreak.domain.Stack;
+import com.sequenceiq.cloudbreak.common.type.AdjustmentType;
 
 public class CloudContext {
 
-    private Long stackId;
-    private String stackName;
-    private String platform;
-    private String variant;
-    private String owner;
-    private String region;
-    private OnFailureAction onFailureAction = OnFailureAction.ROLLBACK;
-    private AdjustmentType adjustmentType;
-    private Long threshold;
-    private Long created;
-
-    public CloudContext(Stack stack) {
-        this.stackId = stack.getId();
-        this.stackName = stack.getName();
-        this.platform = stack.cloudPlatform().name();
-        this.variant = stack.getPlatformVariant();
-        this.region = stack.getRegion();
-        this.owner = stack.getOwner();
-        this.onFailureAction = stack.getOnFailureActionAction();
-        this.created = stack.getCreated();
-        if (stack.getFailurePolicy() != null) {
-            this.adjustmentType = stack.getFailurePolicy().getAdjustmentType();
-            this.threshold = stack.getFailurePolicy().getThreshold();
-        }
-    }
-
-    public CloudContext(Long stackId, String stackName, String platform, String owner) {
-        this.stackId = stackId;
-        this.stackName = stackName;
-        this.platform = platform;
-        this.owner = owner;
-    }
-
-    public CloudContext(Long stackId, String stackName, String platform, String variant, String owner) {
-        this.stackId = stackId;
-        this.stackName = stackName;
-        this.platform = platform;
-        this.variant = variant;
-        this.owner = owner;
+    private final Long stackId;
+    private final String stackName;
+    private final String platform;
+    private final String owner;
+    private final String variant;
+    private final String region;
```

Comment:
remove the followings from CloudContext:
        this.region = params.getRegion();
        this.created = params.getCreated();
        this.adjustmentType = params.getAdjustmentType();
        this.threshold = params.getThreshold();